Doc/library/socket.rst

```diff
@@ -225,6 +225,33 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.9
 
+- :const:`AF_HYPERV` is a Windows-only socket based interface for communicating
```

Comment:
Windows-only? Isn't there Linux support as well?

Reply:
Linux guests on Hyper-V uses AF_VSOCK as per https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/make-integration-service#create-a-hyper-v-socket. This is already present in Python as well luckily.

Reply:
Nice, thanks for the link!

Reply:
You're welcome, thanks for looking at the changes here.